infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/environment/KubernetesEnvironment.java

```diff
@@ -166,9 +168,25 @@ public Builder setConfigMaps(Map<String, ConfigMap> configMaps) {
       return this;
     }
 
+    public Builder setWorkspaceConfigAttributes(Map<String, String> attributes) {
```

Comment:
It is a builder for K8sEnvironment. In other places, these attributes are just attributes but not workspace config attributes. Maybe it would be better to have `setAttributes` here too?

Reply:
OK